src/vs/platform/keybinding/test/common/keybindingsTestUtils.ts

```diff
@@ -7,7 +7,7 @@ import { decodeKeybinding, ResolvedKeybinding } from 'vs/base/common/keybindings
 import { OperatingSystem } from 'vs/base/common/platform';
 import { USLayoutResolvedKeybinding } from 'vs/platform/keybinding/common/usLayoutResolvedKeybinding';
 
-export function createUSLayoutResolvedKeybinding(encodedKeybinding: number, OS: OperatingSystem): ResolvedKeybinding | undefined {
+export function createUSLayoutResolvedKeybinding(encodedKeybinding: number | number[], OS: OperatingSystem): ResolvedKeybinding | undefined {
 	if (encodedKeybinding === 0) {
 		return undefined;
 	}
```

Comment:
I think this is fine (?)

As far as I know `x === 0` is equivalent to `typeof x == "number" && x == 0`, and since `typeof [] === "object"` we'd get the same check. (My understanding / assumption was that TS considers `0` to be a member of the type `number | number[]`, but I don't really know TS in too much detail so that's just a semi-educated guess)